### PR TITLE
Fixuphans

### PR DESCRIPTION
**Pull request by optimumtact**
## Changelog
:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -58110,6 +58102,20 @@
 	pixel_x = 3;
 	pixel_y = 2
 	},
+/obj/merge_conflict_marker{
+	name = "---Merge Conflict Marker---";
+	desc = "A best-effort merge was performed. You must resolve this conflict yourself (manually) and remove this object once complete."
+	},
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:58108]: cough

**Comment by Pickle-Coding** [comment]
Can you explain why this is good for the game?

File: code/modules/reagents/reagent_containers/cups/drinks.dm [hunk 2]
```diff
@@ -150,6 +147,33 @@
...
+	isGlass = FALSE
+	drink_type = BREAKFAST
+
+/obj/item/reagent_containers/cup/glass/coffee/examine(mob/user)
+	. = ..()
+	. += span_notice("Alt-click to toggle cup lid.")
+	return
+
+/obj/item/reagent_containers/cup/glass/coffee/AltClick(mob/user)
+	lid_open = lid_open ? 0 : 1
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/cups/drinks.dm:166]: what are you

File: code/modules/reagents/reagent_containers/cups/drinks.dm [hunk 3]
```diff
@@ -150,6 +147,33 @@
...
+	var/lid_open = 0
+	spillable = TRUE
+	resistance_flags = FREEZE_PROOF
+	isGlass = FALSE
+	drink_type = BREAKFAST
+
+/obj/item/reagent_containers/cup/glass/coffee/examine(mob/user)
+	. = ..()
+	. += span_notice("Alt-click to toggle cup lid.")
+	return
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/cups/drinks.dm:163]: i hate you

File: code/modules/food_and_drinks/machinery/coffeemaker.dm [hunk 4]
```diff
@@ -408,3 +493,251 @@
...
+				else
+					return TRUE
+			else
+				balloon_alert(user, "non-dried beans inside of coffee pack!")
+				return TRUE
+
+	. = TRUE //no afterattack
+
+	update_appearance()
+	return TRUE //no afterattack
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/machinery/coffeemaker.dm:717]: 200w

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (san7890) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:58108]: don't worry CI caught this

**Pull request closed without merging**